go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -521,9 +516,26 @@ func CallEmpty[VerbClient any](ctx context.Context) error {
 	return err
 }
 
+// GetDatabaseHandle returns a database handle using the given database config.
+func GetDatabaseHandle[T ftl.DatabaseConfig]() (ftl.DatabaseHandle[T], error) {
+	reflectedDB := reflection.GetDatabase[T]()
+	if reflectedDB == nil {
+		return ftl.DatabaseHandle[T]{}, fmt.Errorf("could not find database for config")
+	}
+
+	var dbType ftl.DatabaseType
+	switch reflectedDB.DBType {
+	case reflection.DBTypePostgres:
```

Comment:
Do we need separate reflection types? Could we not use the ftl.DatabaseType for this as well?

Reply:
What I mean here is that as we add more types, it'd be great if we didn't have to update N different sets of types, and instead could just update one.

Reply:
yeah, we could just use strings in the reflection type instead of its own enum and translate the string to/from the `ftl.DatabaseType` as needed

the reflection type `reflection.ReflectedDatabaseHandle` can't use the `ftl.DatabaseType` because it can't depend on that package